Movieee/Movieee/UIs/Search Screen/Cell/SearchScreenTableViewCell.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+class SearchScreenTableViewCell: UITableViewCell {
+    
+    @IBOutlet private weak var star1: UIImageView!
+    @IBOutlet private weak var star2: UIImageView!
+    @IBOutlet private weak var star3: UIImageView!
+    @IBOutlet private weak var star4: UIImageView!
+    @IBOutlet private weak var star5: UIImageView!
```

Comment:
```suggestion
    @IBOutlet private var ratingStars: [UIImageView]!
```
em connect tất cả imageView kia vào 1 cái array như này cho tiện dùng